idaes/power_generation/carbon_capture/mea_solvent_system/unit_models/column.py

```diff
@@ -422,53 +414,48 @@ def _make_performance(self):
                         doc='''Integer indexing parameter required for transfer
                              across boundaries of a given volume element''')
-        # Set the integer  indices
+        # Set the integer indices along vapor phase length domain
         for i, x in enumerate(self.vapor_phase.length_domain, 1):
             self.zi[x] = i
 
         # Continuation parameters for initialization
-        self._homotopy_par_m = Param(initialize=0, mutable=True, units=None,
-                                     doc='''Continuation parameter to turn on mass
-                                     transfer terms gradually''')
-        self._homotopy_par_h = Param(initialize=0, mutable=True, units=None,
-                                     doc='''Continuation parameter to turn on heat
-                                     transfer terms gradually''')
-
-        # fixed column pressure
-        if self.config.fix_column_pressure:
-            self.column_pressure = Param(initialize=self.config.column_pressure,
-                                         mutable=True,
-                                         units=pyunits.Pa,
-                                         doc='Fixed operating pressure of column')
-
-        # Interfacial area  parameters
+        self._homotopy_par_m = Param(
+            initialize=0, mutable=True, units=None,
+            doc='Continuation parameter to turn on mass transfer '
+            'terms gradually')
+        self._homotopy_par_h = Param(
+            initialize=0, mutable=True, units=None,
+            doc='Continuation parameter to turn on heat transfer '
+            'terms gradually')
+
+        # Interfacial area parameters
         self.area_interfacial_parA = Var(initialize=0.6486,
                                          units=None,
-                                         doc='''Interfacial area parameter A''')
+                                         doc='Interfacial area parameter A')
 
         self.area_interfacial_parB = Var(initialize=0.12,
                                          units=None,
-                                         doc='''Interfacial area parameter B''')
+                                         doc='Interfacial area parameter B')
         self.area_interfacial_parA.fix(0.6486)
         self.area_interfacial_parB.fix(0.12)
 
         # Holdup  parameters
         self.holdup_parA = Var(initialize=24.2355,
                                units=None,
-                               doc='''holdup parameter A''')
+                               doc='Holdup parameter A')
 
         self.holdup_parB = Var(initialize=0.6471,
                                units=None,
-                               doc='''holdup parameter B''')
+                               doc='Holdup parameter B')
         self.holdup_parA.fix(24.2355)
         self.holdup_parB.fix(0.6471)
 
-
         # Unit Model Variables
         # Geometry
         self.diameter_column = Var(domain=Reals,
```

Comment:
These should be declared before the control volumes - notably, length is required for the control volumes so we should ensure they are consistent.

In fact, we should be creating the length domain at the unit model level and passing this to the control volumes (that is a supported option) to ensure the domains are consistent, and to make it easier to provide custom meshing.